content/1_docs/3_reference/7_plugins/3_components/0_css/component.txt

```diff
@@ -0,0 +1,15 @@
+Title: CSS URLs
+
+----
+
+Text:
+
+```php "/site/plugins/css-helper/index.php"
+Kirby::plugin('my/css', [
+  'components' => [
+    'css' => function (Kirby $kirby, string $url, $options = null): string {
+			return $url . '?' . $fingerprint;
```

Comment:
`$fingerprint` is not defined

Reply:
Sollte nur als Beispiel dienen, mir ist nichts Gutes eingefallen

Reply:
Ja, verstehe ich auch als solches, aber wenn das nicht sehr eindeutig ist, kann das für Nutzer irritierend sein. Unten haben wir ja auch noch mal "YourSnippet::load()`, damit können Anfänger auch nicht allzu viel anfangen, was hier eigentlich erwartet wird.